<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HASHGEN Open Source 0.5</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>851</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Hashes</string>
    </property>
    <widget class="QToolButton" name="btnCopySHA512">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btnCopySHA256">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="fieldSHA256">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>661</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btnCopyMD5">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="fieldMD5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>661</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="fieldSHA512">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>661</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelMD5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MD5</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSHA256">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SHA256</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSHA512">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SHA512</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>File:</string>
    </property>
    <widget class="QToolButton" name="btnLoadFile">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>721</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>841</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File&amp;</string>
    </property>
    <addaction name="acSave"/>
    <addaction name="acSaveAs"/>
    <addaction name="separator"/>
    <addaction name="acQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="acAboutQt"/>
    <addaction name="acAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="acSave">
   <property name="text">
    <string>Save..</string>
   </property>
  </action>
  <action name="acSaveAs">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
  <action name="acQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="acLoadFile">
   <property name="text">
    <string>Load File</string>
   </property>
  </action>
  <action name="acClipMD5">
   <property name="text">
    <string>Copy MD5</string>
   </property>
  </action>
  <action name="acClipSHA256">
   <property name="text">
    <string>Copy SHA256</string>
   </property>
  </action>
  <action name="acClipSHA512">
   <property name="text">
    <string>Copy SHA512</string>
   </property>
  </action>
  <action name="acAboutQt">
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="acAbout">
   <property name="text">
    <string>About..</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
